src/vue/forms/SaleForm/components/FullDescriptionStepForm.vue

```diff
@@ -116,35 +120,40 @@ export default {
     },
   },
 
-  created () {
-    if (this.request) {
-      this.populateForm()
+  async created () {
+    if (+this.former.attrs.requestId) {
+      this.form.youtubeVideo = this.former.attrs.youtubeVideo
+      this.form.fullDescription = await this.getSaleFullDescription(
+        this.former.attrs.saleDescriptionBlobId,
+        this.former.attrs.accountId
+      )
+      this.former.setAttr('fullDescription', this.form.fullDescription)
```

Comment:
need to talk. What is situated this.former.fullDescription? and what happens if we change fullDescription